snap/snapcraft.yaml

```diff
@@ -24,9 +24,15 @@ description: |
     Dump information about the available Vulkan setup.
   - vkcube (Apache-2.0):
     Test the Vulkan setup.
+  - bandwidthTest (MIT):
+    A sample NVIDIA CUDA application can be used to confirm CUDA operational status.
+    (needs nvidia-core22)
+  - deviceQuery (MIT):
+    Query NVIDIA CUDA hardware configuration. (needs nvidia-core22)
 
   This snap uses the graphics-core22 interface to gain support graphics hardware.
   It can be used to verify custom graphics-core22 provider implementations.
+  The CUDA specific apps require NVIDIA capable provider, such as nvidia-core22.
```

Comment:
```suggestion
  The CUDA specific apps require Nvidia capable provider, such as nvidia-core22.
```

Reply:
NVIDIA should always be refered to in all-caps, as per their https://www.nvidia.com/en-us/about-nvidia/legal-info/logo-brand-usage/ guidelines (despite small-caps wordmark in the logo assets).

Reply:
ACK